src/ConfigCatClientOptions.ts

```diff
@@ -215,11 +225,15 @@ export class AutoPollOptions extends OptionsBase implements IAutoPollOptions {
       }
     }
 
-    if (!(this.pollIntervalSeconds >= 1 && (typeof this.pollIntervalSeconds === "number"))) {
+    // https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#maximum_delay_value
+    // https://stackoverflow.com/a/3468650/8656352
+    const maxSetTimeoutIntervalSecs = 2147483.647;
```

Comment:
Please go with 2147483 instead of 2147483.647. We should show this in the documentation too and i think it would look kinda weird there.

Reply:
Fair enough.